map_gui/src/render/lane.rs

```diff
@@ -328,33 +326,119 @@ fn calculate_turn_markings(map: &Map, lane: &Lane) -> Vec<Polygon> {
         return Vec::new();
     }
 
-    // Don't call out the strange lane-changing in intersections. Per target road, find the average
-    // turn angle.
-    let mut angles_per_road: HashMap<RoadID, Vec<Angle>> = HashMap::new();
-    for turn in map.get_turns_from_lane(lane.id) {
-        angles_per_road
-            .entry(turn.id.dst.road)
-            .or_insert_with(Vec::new)
-            .push(turn.angle());
-    }
+    // Only show one arrow per road. They should all be the same angle, so just use last.
```

Comment:
An unexpected bit I noticed is
![Screenshot from 2021-11-22 11-46-28](https://user-images.githubusercontent.com/1664407/142925320-0f58dffc-528e-4168-a1d2-ccf37d8990f4.png)
but that matches how things work before this PR. (It's just something we should reconsider -- it feels weird to be explicit about one lane's turns, but not another.)

Reply:
I agree that having arrows on only some lanes looks weird. Done by [line 311-327](https://github.com/a-b-street/abstreet/blob/5010acbc13ceecf5f06be8a6e8b6d5ea19341de0/map_gui/src/render/lane.rs#L311). That could instead be like if every lane on this road connects to every other possible outbound lane of the same type to provide the behavior you want.